bot/code_review_bot/report/phabricator.py

```diff
@@ -246,9 +246,20 @@ def pluralize(word, nb):
         # Build top comment
         nb = len(issues)
 
+        # Add extra hint when errors are published outside of the patch
+        defects_details = ""
+        if any(
+            issue.is_publishable() and not revision.contains(issue) for issue in issues
+        ):
```

Comment:
We should assert that there is a parent revision here, otherwise it might be confusing if there isn't one (in theory there should be, otherwise where does the issue come from? But you never know, maybe we are missing something...).

See also https://github.com/mozilla/code-review/pull/561#discussion_r434249237.

Reply:
I saw this thread and commented here https://github.com/mozilla/code-review/pull/1243#issuecomment-1197939329.
I don't know much Phabricator API, but I'll look for a way to get this information later on.

Reply:
Yeah when I was writing my comment I hadn't seen your comment yet :)

You can use this function for getting parent information: https://github.com/mozilla/libmozdata/blob/e8c19af0552dfffdff4bfcb69d423a73a3d9f280/libmozdata/phabricator.py#L345.